src/app/productConfig/productConfig.modal.js

```diff
@@ -19,7 +19,7 @@ class ModalInstanceCtrl{
   }
 
   frequencyOrder(f){
-    let order = ['NA', 'MON', 'QUARTERLY', 'ANNUAL', 'SEMIANNUAL'];
+    let order = ['NA', 'MON', 'QUARTERLY', 'SEMIANNUAL', 'ANNUAL'];
```

Comment:
I'm not sure.  I thought I should leave it in just in case.  We may use something like this for gifts already setup as semiannual.  Just new gifts can't be setup this way.